src/Error.ts

```diff
@@ -60,83 +112,69 @@ class StripeError extends Error {
  * CardError is raised when a user enters a card that can't be charged for
  * some reason.
  */
-class StripeCardError extends StripeError {}
+export class StripeCardError extends StripeError {}
```

Comment:
how does the compiled code looks like now for this export?

Reply:
It now looks like this:

```
exports.StripeCardError = StripeCardError;
```